vertx-template-engines/vertx-web-templ-handlebars/src/main/java/io/vertx/ext/web/templ/impl/HandlebarsTemplateEngineImpl.java

```diff
@@ -65,7 +67,14 @@ public void render(RoutingContext context, String templateFileName, Handler<Asyn
           cache.put(templateFileName, template);
         }
       }
-      handler.handle(Future.succeededFuture(Buffer.buffer(template.apply(context.data()))));
+    Context engineContext = Context.newBuilder(context.data()).resolver(
```

Comment:
This list seems to include more resolvers than the default ones + JsonObject + JsonArray. Is there a reason for this? Unless there's a strong motivation, I'd prefer to stick to default ones + JsonObject + JsonArray.